config/has_constexpr_limits_cmd.cpp

```diff
@@ -3,11 +3,13 @@
 //  Boost Software License, Version 1.0. (See accompanying file
 //  LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
 
+// This determines if std::numeric_limits can be constexpr for the compiler/configuration.
+
 #ifndef __GNUC__
-#error "Compiler is not GCC"
+#  error "Compiler is not GCC."
 #endif
 #if __GNUC__ < 9
-#error "Older GCC versions don't support -fconstexpr-ops-limit"
+#  error "Older GCC versions don't support -fconstexpr-ops-limit (aka -fconstexpr-loop-limit=n) ."
```

Comment:
I don't think these are the same at all?  -fconstexpr-loop-limit deals with loops within constexpr functions, where as -fconstexpr-ops-limits sets the total number of integer operations permitted.